extensions_admin/pulp_rpm/extensions/admin/repo_options.py

```diff
@@ -39,6 +39,13 @@ def parse_skip_types(t):
 
     return parsed
 
+
+def parse_string_set(input):
```

Comment:
This should probably live in `parsers` with the other parsers like `parse_boolean`